clevercloud/pre_run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -l
 
-cd secrets-vault
+cd secrets-vault || exit
 git pull
-cd -
-sops -d secrets-vault/c1/$ITOU_ENVIRONMENT.enc.env > .env
+cd - || exit
+sops -d secrets-vault/c1/"$ITOU_ENVIRONMENT".enc.env > .env
```

Comment:
D'ailleurs j'avais jamais fait gaffe mais TIL :
> exit [n]
> The exit status shall be n, if specified. Otherwise, the value shall be the exit value of the last command executed, or zero if no command was executed.